doc/contrib/release.rst

```diff
@@ -75,6 +75,8 @@ Release
    
 #. Change ``VERSION``. Make sure does not end with ``-rc`` or ``-dev``.
 
+#. Update ``scripts/check-all-tags.sh`` to check the current tag.
```

Comment:
Automated testing will have caught any changes by now, so all that's left is to add nextver to the loops, right?

Reply:
Yes, just add `tag-hash`. The check-all script is for our manual regression testing, it's not used in an automated workflow.